core/node/da_clients/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 pub mod avail;
 pub mod celestia;
-pub mod eigen;
+//pub mod eigen;
```

Comment:
this is because it wasn't compiling successfully. if it's fixed now, feel free to uncomment.